packages/nextjs/utils/scaffold-eth/networks.ts

```diff
@@ -37,9 +152,13 @@ export const NETWORKS_EXTRA_DATA: Record<string, ChainAttributes> = {
   },
   [chains.optimismGoerli.id]: {
     color: "#f01a37",
+    etherscanEndpoint: "https://api-goerli-optimism.etherscan.io",
+    etherscanApiKey: MAIN_ETHERSCAN_API_KEY,
```

Comment:
Endpoint and ApiKey are working fine (which is strange, because Optimistic API is working also for this endpoint).

In [docs](https://docs.optimism.etherscan.io/getting-started/endpoint-urls) they referencing `https://api-goerli-optimistic.etherscan.io` as endpoint, should we change it and also use Optimistic Etherscan API to make it more consistant?